vizro-core/src/vizro/models/_components/form/_text_area.py

```diff
@@ -0,0 +1,49 @@
+from typing import List, Literal
+
+import dash_bootstrap_components as dbc
+from dash import html
+
+try:
+    from pydantic.v1 import Field
+except ImportError:  # pragma: no cov
+    from pydantic import Field
+
+from vizro.models import Action, VizroBaseModel
+from vizro.models._action._actions_chain import _action_validator_factory
+from vizro.models._models_utils import _log_call
+
+
+class TextArea(VizroBaseModel):
+    """Component provided to `Form` to allow user multi-line text input.
+
+    Args:
+        type (Literal["text_area"]): Defaults to `"text_area"`.
+        title (str): Title to be displayed. Defaults to `""`.
+        placeholder (str): Default text to display in input field. Defaults to `""`.
+        actions (Optional[List[Action]]): Defaults to `[]`.
+    """
+
+    type: Literal["text_area"] = "text_area"
+    title: str = Field("", description="Title to be displayed")
```

Comment:
```suggestion
    # TODO: before making public consider naming this field (or giving an alias) label instead of title
    title: str = Field("", description="Title to be displayed")
```